guides/sdk-docs/public-apis/app-tokens.md

```diff
@@ -65,4 +65,44 @@ If you need access to make API calls on behalf of the user without having to wai
   },
 ```
 
-When a user installs an app we will call the specified URL with a client ID and Secret that can be used to retrieve an access token to make calls on behalf of the user.
+When a user installs an app we will call the specified URL with a client ID and Secret that can be used to retrieve an access token to make calls on behalf of the user. The message will look something like this:
+
+```JSON
+{
+  "type": "INSTALLED",
+  "irisAppId": "@organization/appname",
+  "accountId": "6b5cb246-7945-4bb6-bec5-2a1413ffd1b3",
+  "clientId": "0oa10o3h43eulvWtz358",
+  "clientSecret": "wZtQppu1huik4cYBAhYXnZS29pSudTBne2SE6WEM5C5"
+}
+```
+
+The `accountId` identifies which account installed the app and should be stored securely together with the client ID and secret.
+
+To retrieve an access token for our API you should call `https://auth.trackunit.com/apptoken` like this:
+
+```sh
+curl --location --request POST 'https://auth.trackunit.com/apptoken' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--data-urlencode 'grant_type=client_credentials' \
+--data-urlencode 'client_id=0oa10o3h43eulvWtz358' \
+--data-urlencode 'client_secret=wZtQppu1huik4cYBAhYXnZS29pSudTBne2SE6WEM5C5' \
+--data-urlencode 'scope=asset.view'
```

Comment:
What is the format if i need to add more than one scope?

Reply:
Expanded example with 2 scopes